FWCore/ParameterSet/python/Config.py

```diff
@@ -27,6 +27,14 @@
 if sys.getrecursionlimit()<5000:
     sys.setrecursionlimit(5000)
 
+class edm(object):
+    class error(object):
+        #Allowed errors to used within Python
+        Configuration = "{Configuration}"
+
+class EDMException(Exception):
+    pass
```

Comment:
It is not crystal clear to me how these would be used. Based on the implementation I'd guess
```py
raise EDMException(edm.error.Configuration + "Real exception message")
```

How about something along the following interface?
```py
raise EDMException(edm.error.Configuration, "Real exception message")
```